src/main/kotlin/blackJack/domain/Deck.kt

```diff
@@ -4,21 +4,23 @@ class Deck() {
     var cards = DEFAULT_DECK
         private set
 
-    fun shuffle(shuffleDeck: (cards: List<Card>) -> List<Card>) {
-        cards = shuffleDeck(cards)
+    fun getFirstCard(): Card {
+        val card = cards[FIRST_CARD]
+        deleteFirstCard()
+        return card
     }
 
-    fun getCard(): Card {
-        val card = cards[0]
-        deleteCard()
-        return card
+    private fun deleteFirstCard() {
+        cards = cards.drop(DELETE_CARD)
     }
 
-    private fun deleteCard() {
-        cards = cards.drop(1)
+    fun shuffle(cards: List<Card>) {
+        this.cards = cards.shuffled()
     }
 
     companion object {
         val DEFAULT_DECK = Shape.values().flatMap { shape -> Denomination.values().map { Card(shape, it) } }
+        const val FIRST_CARD = 0
```

Comment:
`Deck` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
        private const val FIRST_CARD = 0
```